Tests/SwiftParserTest/DeclarationTests.swift

```diff
@@ -1427,6 +1427,125 @@ final class DeclarationTests: XCTestCase {
     )
   }
 
+  func testAttributedMacroExpansionDeclaration() {
```

Comment:
Could add a test when split by a newline rather than space? And also one for with both attributes and modifiers?